metabase/metabase
# Browsing nav cleanup

Fixes up the alignment of nav items in the new nav sidebar. This also moves the less useful links for "Archive" and "Other users personal collections" out of the way into a more menu next to the collection section heading. While we're add it, why not include a new icon for the open / close sidebar trigger.

This also tries a softer selected state and nav item styling so that if you do have the sidebar open, it doesn't dominate as much visually and the content continues to dominate. I'm partially worried it's too soft from a contrast perspective so that's something we might try and tweak over time.

<img width="1392" alt="Screen Shot 2022-03-31 at 4 59 43 PM" src="https://user-images.githubusercontent.com/5248953/161284737-5b975cfd-80b0-49cf-8e67-6a3e9bb89242.png">

How to test
- Fire up the branch and open up the sidebar.
- All your collections, bookmarks, and the browse data link should be aligned and spaced the same regardless of section.
